pkg/kubernetes/client_test.go

```diff
@@ -325,9 +325,14 @@ var _ = Describe("Test Namespace KubeController Methods", func() {
 
 			_, err := kubeClient.CoreV1().Services(testNamespaceName).Create(context.TODO(), svc, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
+
+			// Wait for both pods and the service to be created
+			<-kubeController.GetAnnouncementsChannel(Pods)
+			<-kubeController.GetAnnouncementsChannel(Pods)
```

Comment:
you have two announcement channels for the pods now

Reply:
yes, because the test creates 2 pods.